### ISSUE TITLE
Artifact appears as the training goes on

### ISSUE THREAD

**Issue by adelebei** [prstrive/EPCDepth#6]
Hi, dear author, I really appreciate your awesome work! It is more stable and performs better than depth estimation with monocular video.

However, I met a problem when I trained EPCNet on my own dataset. 
When the model is only trained for 3 epochs, the performance is good. However, when i trained for more epochs (such as 20 epochs), artifacts appears on the predicted disparity map, as shown in the following figures.
![image](https://user-images.githubusercontent.com/39139200/143849217-befdc292-50dc-41df-a65f-5e7e7f8a25be.png)
![image(1)](https://user-images.githubusercontent.com/39139200/143849228-b81c75fb-aab5-4d26-ae0a-c6951cd7e345.png)

What could be possible to lead to the result? Could you provide me some advice?
THANK YOU!

**Comment by prstrive** [comment]
You may try to reduce the learning rate in advance (e.g. divided by 10 every 5 epochs). Besides, the spp-distillation module may lead a wrong supervised signal in low-textured regions and other undesirable areas, because these areas do not meet our assumption that the smaller the reconstruction error, the better the depth estimation. So you can consider turning off this module.

**Comment by adelebei** [comment]
Thank you for your advice! I will try turning off the SPP-distillation module.
